invenio_edugain/views.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 Graz University of Technology.
+#
+# invenio-edugain is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""invenio-edugain views."""
+
+from flask import Blueprint, render_template
+
+from .utils import get_idp_data_dict
+
+blueprint = Blueprint(
```

Comment:
please use the `create_blueprint()` approach:

```
def create_blueprint(app):
    routes = app.config["EDUGAIN_ROUTES"]
    blueprint = Blueprint(...)
    
    blueprint.add_url_rule(routes["discover"], view_func=login_discover)
```

Reply:
Done